src/ansys/mapdl/core/math.py

```diff
@@ -902,7 +902,18 @@ def sparse(self, mat, thresh="", **kwargs):
         kwargs.setdefault("mute", True)
         self._mapdl.run(f"*COMP,{mat.id},SPARSE,{thresh}", **kwargs)
 
-    def eigs(self, nev, k, m=None, c=None, phi=None, algo=None, fmin=None, fmax=None):
+    def eigs(
+        self,
+        nev,
+        k,
+        m=None,
+        c=None,
+        phi=None,
+        algo=None,
+        fmin=None,
+        fmax=None,
+        cpxmod=None,
```

Comment:
We should probably add something like:

```py
if not cpxmod:
    cpxmod = ""
```
So you pass an empty string by default to the MAPDL instance. An empty string in an MAPDL argument means you will be using the default value.